rasa/shared/nlu/training_data/formats/rasa.py

```diff
@@ -49,9 +94,20 @@ def read_from_json(self, js: Dict[Text, Any], **_: Any) -> "TrainingData":
 
 
 class RasaWriter(TrainingDataWriter):
+    """Dumps NLU data as Rasa JSON string."""
+
+    def __init__(self) -> None:
+        """Creates writer."""
+        super().__init__()
+        rasa.shared.utils.io.raise_deprecation_warning(
+            "NLU data in Rasa JSON format is deprecated and will be removed in Rasa "
+            "Open Source 4.0.0. Please convert your JSON NLU data to the "
+            "Rasa YAML format.",
```

Comment:
Should we also add here or whenever the `rasa data convert nlu` command is used that the command itself will be removed in 4.0?